WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1614,35 +1613,6 @@ public void onEventMainThread(ReaderEvents.SearchPostsEnded event) {
         }
     }
 
-    /*
-     * called when user taps follow item in popup menu for a post
-     */
-    private void toggleFollowStatusForPost(final ReaderPost post) {
```

Comment:
I would have to update this `ReaderBlogActions.followBlogForPost` but this whole method was unused so I simply removed it